File: code_examples/3-Structured_Problem_Solving/19-Loops_and_Ifs/CPP/main.cpp
// Copyright (c) 2020 St. Mother Teresa HS All rights reserved.
//
// Created by: Mr. Coxall
// Created on: Sep 2020
// This program is the Fizz Buzz program

#include <iostream>

int main() {
    // this function is the Fizz Buzz program
    for (int counter = 1; counter <= 100; counter++) {
        if (counter % 3 == 0 && counter % 5 == 0) {
            std::cout << "Fizz-Buzz" << std::endl;
        } else if (counter % 3 == 0) {
            std::cout << "Fizz" << std::endl;
        } else if (counter % 5 == 0) {
            std::cout << "Buzz" << std::endl;
        }  else {
            std::cout << counter << std::endl;
        }
    }

    std::cout << "\nDone." << std::endl;
    return 0;
}

File: code_examples/3-Structured_Problem_Solving/19-Loops_and_Ifs/CSharp/main.cs
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program is the Fizz Buzz program
*/

using System;

/*
 * The Program class
*/
class Program {
    static void Main() {
        // this function is the Fizz Buzz program
        for (int counter = 1; counter <= 100; counter++) {
            if (counter % 3 == 0 && counter % 5 == 0) {
                Console.WriteLine("Fizz-Buzz");
            } else if (counter % 3 == 0) {
                Console.WriteLine("Fizz");
            } else if (counter % 5 == 0) {
                Console.WriteLine("Buzz");
            } else {
                Console.WriteLine(counter);
            }
        }

        Console.WriteLine("\nDone.");
    }
}